pkg/xds/bootstrap/types/bootstrap_request.go

```diff
@@ -23,6 +23,14 @@ type BootstrapRequest struct {
 	SystemCaPath         string            `json:"systemCaPath"`
 }
 
+type XdsMode string
```

Comment:
We are using "XdsTransportProtocolVariant" at many places, so why not use "Variant" here to replace "Mode"?